src/olympia/blocklist/views.py

```diff
@@ -111,15 +125,25 @@ def get_items(apiver, app, appver=None):
     return items, details
 
 
-def get_plugins(apiver, app, appver=None):
+def get_plugins(apiver=3, app=None, appver=None):
     # API versions < 3 ignore targetApplication entries for plugins so only
     # block the plugin if the appver is within the block range.
+
+    if app:
+        app_query = (Q(app__isnull=True) |
+                     Q(app__guid=app) |
+                     Q(app__guid__isnull=True))
+    else:
```

Comment:
Oh, I see now, and I was wrong, it would work... I hope there is a better way... I'll try a few things locally.